src/ray/common/ray_config_def.h

```diff
@@ -42,6 +42,10 @@ RAY_CONFIG(uint64_t, num_heartbeats_warning, 5)
 
 /// The duration between reporting resources sent by the raylets.
 RAY_CONFIG(uint64_t, raylet_report_resources_period_milliseconds, 100)
+/// For a raylet, if the last resource report was sent more than this many
+/// report periods ago, then a warning will be logged that the report
+/// handler is drifting.
+RAY_CONFIG(uint64_t, num_resource_report_periods_warning, 5)
```

Comment:
Is it 500ms now? I think it is probably too small (I've seen this can increase to 1 second pretty frequently). Why don't we set 50?

Reply:
Hmm I'm not sure about that. We should be setting this to a threshold that indicates raylet pressure severe enough to cause negative externalities, like very stale scheduling decisions or the creation of task submission/execution bottlenecks within this raylet, which I think would happen well before 5 seconds, right? I'd rather set this to e.g. 10 (1 second interval) and aggressively mitigate the issues causing such raylet pressure.

Reply:
ah yeah I forgot that this is just a warning and not crashing the raylet.

Reply:
Does a 10x multiplier sound good, or is the current 5x multiplier fine?

Reply:
Ah just saw your approval of the PR, never mind! 😁

Reply:
Yeah I think 5 is fine! I think this might match the previous behavior too :)